src/Glpi/Form/QuestionType/QuestionTypeInterface.php

```diff
@@ -133,6 +133,16 @@ public function renderEndUserTemplate(Question $question): string;
      */
     public function renderAnswerTemplate($answer): string;
```

Comment:
If the type is mixed, you should still add the corresponding type hinting. It will prevent any code quality tool to detect it as a missing type hinting.
```suggestion
    public function renderAnswerTemplate(mixed $answer): string;
```